src/Components/Bookings.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "../data/fakeBookings.json";
+
+const Bookings = () => {
+  const search = searchVal => {
+    console.info("TO DO!", searchVal);
+  };
+  const [bookings, setBookings] = useState([]);
```

Comment:
Great use of React hooks and fetch method!👏 As you're fetching your data now, there is no need to import it on line 4 any more. Also the convention is to keep the state at the top of the component, so I would move it to line 7

Reply:
done